composer.json

```diff
@@ -14,7 +14,7 @@
     "ext-json": "*",
     "ext-libxml": "*",
     "ext-spl": "*",
-    "ampproject/amp-toolbox": "0.11.2",
+    "ampproject/amp-toolbox": "dev-main",
```

Comment:
@westonruter Is it intended to bump it to `dev-main`?

Reply:
Yes, in order to update php-toolbox-php to the version that adds PHP 8.2 support: https://github.com/ampproject/amp-toolbox-php/pull/533